src/main/cpp/subsystems/Chassis.cpp

```diff
@@ -13,6 +15,7 @@ std::shared_ptr<Chassis> Chassis::getInstance() {
   return self;
 }
 
+std::shared_ptr<NetworkTable> table = nt::NetworkTableInstance::GetDefault().GetTable("limelight");
```

Comment:
`table` cannot go here. Move table to a member variable (where it should be named `mLimelightTable` (or something similar)) of `Chassis` and initialize in the constructor.